dockers/docker-database/Dockerfile.j2

```diff
@@ -35,6 +35,15 @@ RUN sed -ri 's/^(save .*$)/# \1/g;                       \
              s/^client-output-buffer-limit pubsub [0-9]+mb [0-9]+mb [0-9]+/client-output-buffer-limit pubsub 0 0 0/ \
             ' /etc/redis/redis.conf
 
-COPY ["supervisord.conf", "/etc/supervisor/conf.d/"]
-
-ENTRYPOINT ["/usr/bin/supervisord"]
+RUN sed -ri 's/DAEMON_ARGS=.*/DAEMON_ARGS=\/etc\/redis\/\`basename ${0}\`.conf/;        \
```

Comment:
>/ [](start = 14, length = 1)

You can use other delimiters to make code clean. ref: [https://backreference.org/2010/02/20/using-different-delimiters-in-sed/](https://backreference.org/2010/02/20/using-different-delimiters-in-sed/)

Reply:
this is fixed